tasks/prime.py

```diff
@@ -347,3 +347,34 @@ def create_move_task_order(self):
 
         if success:
             self.set_prime_data(PrimeObjects.MOVE_TASK_ORDER, mto_data)
+
+    @tag(PrimeObjects.MTO_SERVICE_ITEM.value, "updateMTOServiceItemStatus")
+    @task
+    def update_mto_service_item_status(self):
+        mto_shipment = self.get_random_data(PrimeObjects.MTO_SHIPMENT)
+        # if we don't have an mto shipment we can't run this task
+        if not mto_shipment:
+            return
+
+        # if we don't have any service items created for mto shipment can't run this task
+        if mto_shipment.get("mtoServiceItems") is None:
+            return
```

Comment:
Is there a reason you aren't grabbing a MTO_SERVICE_ITEM to start with? I don't think there's a reason to have the shipment around too. It seems like the issues with `create_mto_service_items` have been mostly fixed - or maybe I've been lucky with successful tests that haven't used the bad options haha

Reply:
Hey thanks for this comment. I had more success grabbing the `mto_shipment` first. Originally, when I implemented this, it look more like:
```
    mto_service_item = self.get_random_data(PrimeObjects.MTO_SERVICE_ITEM)
        # if we don't have an mto shipment we can't run this task
        if not mto_service_item:
            return

        payload = self.fake_request("/service-items/{mtoServiceItemID}/status", "patch")
        headers = {"content-type": "application/json", "If-Match": mto_service_item["eTag"]}

        resp = self.client.patch(
            support_path(f"/service-items/{mto_service_item['id']}/status"),
            name=support_path("/service-items/:mtoShipmentID/status"),
            data=json.dumps(payload),
            headers=headers,
            **self.user.cert_kwargs,
        )

        mto_service_item_data, success = check_response(resp, "Update MTO service item status")

        if success:
            self.replace_prime_data(PrimeObjects.MTO_SHIPMENT, mto_service_item, mto_service_item_data)
```
As you can see from above, this grabs the `mto_service_item` initially. The problem with this implementation is that it results in more `409` status codes. When implementing this load test as you suggest, I run into more failures. On average, 20-40% of the calls fail whereas with the current implementation as I've proposed we have 100% success rate. I'd rather keep it the way it is given that we have a higher success rate. Please let me know if there's something I could be doing to get a higher success rate. See below: 
![Screen Shot 2020-12-16 at 4 55 56 PM](https://user-images.githubusercontent.com/7574207/102411374-d6cd1500-3fbf-11eb-983f-6b0e381d4a37.png)

Reply:
Take a look at the success rates. 
![Screen Shot 2020-12-16 at 5 04 00 PM](https://user-images.githubusercontent.com/7574207/102411929-c4071000-3fc0-11eb-899c-c7ba0090da18.png)

Reply:
hmm, I'm not sure if this is a copy paste error now, but at the bottom you're replacing the info for a MTO_SHIPMENT, not a MTO_SERVICE_ITEM. This means the old service items with the old eTags would still be in the list. Is it possible that was causing your problem?

Reply:
I also think some of these errors 409 are expected, and there's nothing preventing this specific error from happening by grabbing a service item from a shipment (a shipment can also have an APPROVED service item that the endpoint tries to APPROVE again). I suspect that if you ran both versions for a sufficiently long period of time, the rate of success would even out. There's the random element at play too

Reply:
Ok! Great! I've updated this to grab the `mto_service_item` instead of the `mto_shipment` and updated the `replace_prime_data` call to replace the `MTO_SERVICE_ITEM` instead of the `MTO_SHIPMENT`.